cpp/include/cudf/detail/segmented_reduction.cuh

```diff
@@ -0,0 +1,200 @@
+/*
+ * Copyright (c) 2022-2023, NVIDIA CORPORATION.
```

Comment:
With this being a new file, should it be 2023 only or is the copyright starting further back since this code was moved? If it is dated on code creation, should it go back to match the original source?

Reply:
I don't think it matters too much as long as it has the current year.